test/jdk/java/awt/dnd/RejectDragDropActionTest.java

```diff
@@ -74,27 +73,31 @@ public void drop(DropTargetDropEvent dtde) {
         };
     private final DropTarget dt = new DropTarget(frame, dtl);
 
-    public static void main(String[] args) throws InterruptedException,
-            InvocationTargetException, AWTException {
+    public static void main(String[] args) throws Exception {
         EventQueue.invokeAndWait(() -> {
             frame = new Frame("RejectDragDropActionTest");
             ds = DragSource.getDefaultDragSource();
             dgl = dge -> dge.startDrag(null, new StringSelection("OOKK"));
-            dgr = ds.createDefaultDragGestureRecognizer(frame, DnDConstants.ACTION_COPY, dgl);
-            frame.setBounds(100, 100, 200, 200);
+            dgr = ds.createDefaultDragGestureRecognizer(frame,
+                    DnDConstants.ACTION_COPY, dgl);
+            frame.setPreferredSize(new Dimension(200, 200));
+            frame.pack();
```

Comment:
```suggestion
            frame.setSize(200, 200);
```
frame.pack() not required as frame size is explicitly specified.

Reply:
Good call. Was testing out different changes, and this was leftover. Modified back to setSize. Thanks